docs/workloads/azure/data/getting_started/core_data_platform_deployment_azure.md

```diff
@@ -7,12 +7,94 @@ hide_table_of_contents: false
+required for setting up Ensono Digital Stacks. It will then save this configuration out to a file that can be
+read in using the scaffold command.
+
+```cmd
+stacks-cli interactive
+```
+
+The majority of the questions are self-explanatory; please refer to the **Stacks.CLI.Manual** for further detail, however the following two will define the type of the target project. 
+
+| Question                                      | Required value for data project |
+|-----------------------------------------------|---------------------------------|
+| What framework should be used for the project?| infra                           |
+| Which type of infrastructure is required?     | data                            |
+
+The resulting configuration file named `stacks.yml` contains all of the configuration that was used to generate the project,
+which means it can be used to produce the same project stack again.
+
+The CLI can be used with a configuration file to generate the Ensono Digital Stacks based projects using `stacks-cli scaffold`.
+
+```cmd
+stacks-cli scaffold -c ./stacks.yml
+```
+
+Open the project locally and push the generated project to the target remote repository's `main` branch.
+
+## Step 2: Add Infrastructure pipeline in Azure DevOps
+
+Open the project locally and create a new feature branch e.g.:
+
+```bash
+git checkout -b feat/infra-pipeline
+```
+
+YAML file containing a template Azure DevOps CI/CD pipeline for building and deploying the core infrastructure, named `air-infrastructure-data.yml` is configured in the `build/azDevOps/azurefolder`
+
+YAML file `air-infrastructure-data.yml` should be added as the definition for a new pipeline in Azure DevOps.
+
+1. Sign-in to your Azure DevOps organization and go to your project
+2. Go to Pipelines, and then select **New pipeline**
+3. Name the new pipeline, e.g. `amido.stacks-data-infrastructure`
+4. For the pipeline definition, specify the YAML file in the project repository feature branch (`air-infrastructure-data.yml`) and save
+5. The new pipeline will require access to any Azure DevOps pipeline variable groups specified in the pipeline YMAL. Under each variable group, go to 'Pipeline permissions' and add the pipeline.
+
+The `build/azDevOps/azure/network` folder contains a YAML file named `air-infrastructure-data-network.yml` that serves as a template for Azure DevOps CI/CD pipeline for networking and self-hosted agent infrastructure. If the client or user does not have their own network and self-hosted agent, they must establish the networking pipeline in Azure DevOps following the procedures outlined above. This setup should be completed before initiating the core infrastructure pipeline. The `build/azDevOps/azure/network` folder also includes another YAML file called `air-infrastructure-data-network-vars.yml` that contains the variables used in the networking pipeline. These variables must be updated as well as the project requirements.
```

Comment:
Could add a subheading before this part, e.g. 'Optional: Networking and self-hosted agent'

Reply:
updated